corehq/apps/smsbillables/models.py

```diff
@@ -312,26 +321,40 @@ def create(cls, message_log, api_response=None):
         country_code, national_number = get_country_code_and_national_number(phone_number)
 
         if backend_instance is None or _sms_backend_is_global(backend_instance):
-            billable.gateway_fee = SmsGatewayFee.get_by_criteria(
-                backend_api_id,
-                direction,
-                backend_instance=backend_instance,
-                country_code=country_code,
-                national_number=national_number,
-            )
-            if billable.gateway_fee is not None:
-                conversion_rate = billable.gateway_fee.currency.rate_to_default
+            if backend_instance is None and backend_api_id == TwilioBackend.get_api_id():
+                def _get_twilio_client():
+                    twilio_backend = MobileBackend.load_by_name(None, "MOBILE_BACKEND_TWILIO")
```

Comment:
i think you can move forward with this pr, and assume that going forward each sms log with a twilio api id will have the backend_id set appropriately (and we'll just hold off on merge until my change gets in next week).  so sometimes the backend_id might also refer to a project-level gateway (backend.is_global == False) in which case we'd want to skip charging them the gateway fee.